frontend/src/devicelist/DevicesTable.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+    Box,
+    Table,
+    TableHead,
+    TableRow,
+    TableCell,
+    TableContainer,
+    TableBody,
+    Typography
+} from "@material-ui/core"
+
+import SingleTableRow from './SingleTableRow'
+import HeadCells from './HeadCells'
+
+export default class DevicesTable extends React.Component {
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
+        return (
+            <React.Fragment>
+                <Box>
+                    <TableContainer style={{ maxHeight: 500 }}>
+                        <Table stickyHeader>
```

Comment:
I'm not sure why `stickyHeader` isn't actually making the header sticky? Any idea why? Also remember to add `aria-label="sticky table"` after `stickyHeader` for accessibility reasons after finding a way to make it sticky.

Reply:
been looking at this for a while... i think we're better off writing a bug report for it and leaving it as-is for the moment. it's a bit inconvenient but unfortunately it's not a quick fix :(

Reply:
Bug report here: #106